### ISSUE TITLE
render issue with long tables

### ISSUE THREAD

**Issue by erfanium** [PejmanNik/jikji#1]
Hi!
Today, I tried this package and find out there's some render problems on rendering long tables

Jikji output:
![Screenshot from 2022-12-03 20-50-18](https://user-images.githubusercontent.com/47688578/205453612-3ef31129-4d33-433e-b267-4bdf8eac2c97.png)

Browser print page output (correct):
![Screenshot from 2022-12-03 20-54-40](https://user-images.githubusercontent.com/47688578/205453642-a5565ece-671e-4320-90f2-9b35b818d9fa.png)

Problems:
1- table's header and footer have not been repeated on every page.
2- table's column width are difference in the first page part and second page part. 

I'm able to contribute to this project, but I need help, I don't know how this module internally works. can you. can you give me a clue?

Thanks for making this great tool

**Comment by PejmanNik** [comment]
I tried to reproduce the overflow bug, but It wasn't successful, so I need that simple react component file from you.

![image](https://user-images.githubusercontent.com/18081640/205459972-0dd6974f-c778-4bf9-bf76-bc454b36d310.png)

**Comment by erfanium** [comment]
@PejmanNik 
The first issue was not about overflow issue. let me explain it a bit more:
for rendering long tables, we should repeat the header and footer of the table for each page. let's say we have this html:
```html
<table>
  <thead>
    This is table header
  </thead>
  
  <tbody>
    Row 1
    Row 2
    Row 3
    ...
    Row 99
  </tbody>
  
  <tfoot>
    This is table footer
  </tfoot>
</table>

```

The render result should be like:
Page 1:
```
This is table header
row 1
row 2
row 3
This is table footer
```
Page 2:
```
This is table header
row 4
row 5
row 6
This is table footer
```

and so on

**Comment by erfanium** [comment]
> unfortunately, the library doesn't have many usages.

Our company delinternet.com can be the first one :-)
We have the need for highly customize-able automatic report/document system but we didn't find anything useful.

**Comment by PejmanNik** [comment]
Oh, yes I got your issues. just biased the table problems I saw and overflow in the first-page footer and the table borders.
I'll try to create simple Table component that cover basic needs, then it can help you to extend it if you need more customization.

**Comment by PejmanNik** [comment]
It is working now, but I need to do more tests, hopefully, I can manage to release it until next weekend.
For problem No 2, we need to keep the column width after the first mount and use it in the next mount/renders.

![image](https://user-images.githubusercontent.com/18081640/205497149-d123c3c3-3d35-4b11-b980-114b5374f4b6.png)

**Comment by PejmanNik** [comment]
It should be fixed with version `0.2.0`. Just wrap your HTML `table` component with the `Table` component; more info: [Table](https://jikji.xyz/docs/components/utilize/table)

Feel free to reopen it if it doesn't fix the issue.

**Comment by erfanium** [comment]
@PejmanNik thanks man
I'll try it today
